packages/react-wildcat-handoff/src/utils/serverRender.js

```diff
@@ -134,7 +134,7 @@ function getHtmlNotFoundTemplate(serverSettings) {
     if (htmlNotFoundTemplate) {
         return {
             status: 404,
-            html: htmlNotFoundTemplate
+            html: htmlNotFoundTemplate()
```

Comment:
@jamsea See line 102 of https://github.com/nfl/react-wildcat/blob/master/packages/react-wildcat-handoff/test/serverHandoffSpec.js